main.py

```diff
@@ -40,7 +43,16 @@ def main():
                 df = import_a_csv_file()
             case "11":
                 break
-
+            case "1":
```

Comment:
Can you add `case "1"` and `case "2"` above `case "11"` ?
The numbers should be sorted.